ui/pages/Project/components/FamilyTable/IndividualRow.jsx

```diff
@@ -186,6 +186,14 @@ const DataDetails = React.memo(({ loadedSamples, individual, mmeSubmission }) =>
         />
       ) : <MmeStatusLabel title="Submitted to MME" dateField="lastModifiedDate" color="violet" individual={individual} mmeSubmission={mmeSubmission} />
     )}
+    {(individual.hasRnaOutlierData || individual.hasRnaSpliceOutlierData) && (
+      <Link
```

Comment:
don't you need this to replace the existing link, not just add a new one?

Reply:
Yes, I'll remove the old one after the new one starts working.